rotkehlchen/tests/api/test_addressbook.py

```diff
@@ -424,10 +431,12 @@ def test_delete_addressbook(
     db_addressbook = DBAddressbook(rotkehlchen_api_server.rest_api.rotkehlchen.data.db)
     with db_addressbook.write_ctx(book_type=book_type) as write_cursor:
         db_addressbook.add_addressbook_entries(write_cursor=write_cursor, entries=generated_entries)  # noqa: E501
-    addresses_to_delete = [
-        to_checksum_address('0x9d904063e7e120302a13c6820561940538a2ad57'),
-        to_checksum_address('0x3D61AEBB1238062a21BE5CC79df308f030BF0c1B'),
-    ]
+
+    address_a = string_to_evm_address('0x9D904063e7e120302a13C6820561940538a2Ad57')
```

Comment:
you are repeating the same address again here. Just save it in a module-scoped variable.

Same for other address you seem to repeat like this